packages/browser-wallet/src/channel-wallet.ts

```diff
@@ -242,58 +244,59 @@ export class ChannelWallet {
    *  START of wallet 2.0
    */
 
-  // TODO: an event should be associated with a single objective. In other words, this function should not
-  //        map an event to all stored objectives
-  private async crankRichObjectives(event: DirectFunder.OpenChannelEvent): Promise<void> {
-    const richObjectives = this.store.richObjectives;
-    for (const channelId of Object.keys(richObjectives)) {
-      const richObjective = richObjectives[channelId];
-      const result = DirectFunder.openChannelCranker(
-        richObjective,
-        event,
-        await this.store.getPrivateKey(await this.store.getAddress())
-      );
-
-      // TODO: this should invoke a store method
-      richObjectives[channelId] = result.objective;
-
-      for (const action of result.actions) {
-        switch (action.type) {
-          case 'sendStates':
-            await Promise.all(action.states.map(state => this.store.addState(state, true)));
-            break;
-          case 'deposit':
-            const fundingMilestones = DirectFunder.utils.fundingMilestone(
-              richObjective.openingState,
-              richObjective.openingState.participants[richObjective.myIndex].destination
-            );
-
-            const txHash = await this.store.chain.deposit(
-              channelId,
-              fundingMilestones.targetBefore,
-              action.amount
-            );
-            this.crankRichObjectives({
-              type: 'DepositSubmitted',
-              tx: txHash ?? '',
-              attempt: 0,
-              submittedAt: Date.now()
-            });
-
-            break;
-          default:
-            throw new Error('Not expected to reach here');
-        }
+  private async crankRichObjective(event: RichObjectiveEvent): Promise<void> {
```

Comment:
It seems that there are some whitespace changes here. Could we run `prettier` against all files in `browser-wallet` in a separate PR?

Reply:
A loop was removed from `crankRichObjective`. So the diff appears as a whitespace change.

That being said, it is not a bad idea to run prettier anyways!

Reply:
You can also hide whitespace changes.
- there's a UI way to do it
- or you can add the `&w=1` param. ([link](https://github.com/statechannels/statechannels/pull/3511/files?diff=split&w=1))

Reply:
Good point! I think I noticed some whitespace changes somewhere else as well but it's quite possible I'm mistaken.